src/style.css

```diff
@@ -0,0 +1,129 @@
+@font-face {
+  font-family: Avenir-Book;
+  src: url(/fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(/fonts/Avenir-Heavy.ttf);
+  font-weight: 300;
+}
+
+body {
+  margin: 0 auto;
+}
+
+.big__form {
+  position: relative;
+  box-sizing: border-box;
+  width: calc(100% - 50px);
+  height: 70px;
+  left: 50px;
+  top: 20px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  background: #fff;
+}
+
+.big__form::before {
+  display: block;
+  position: absolute;
+  content: "";
+  width: 19px;
+  height: 19px;
+  left: 26px;
+  top: 25px;
+  background: url(/images/Search.svg);
+}
+
+.big__form .form__input {
+  position: absolute;
+  width: 158px;
+  left: 62px;
+  top: 35.71%;
+  bottom: 37.14%;
+  border: none;
+  font-family: Avenir-Heavy, sans-serif;
+}
+
+.big__form .form__input::placeholder {
+  font-family: Avenir-Book, Arial, sans-serif;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+}
+
+.small__form {
```

Comment:
too many repeating styles, go through [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md) once again and pay attention to 2nd item